test/lib/actions/ConfigAction.test.ts

```diff
@@ -409,7 +481,6 @@ describe('ConfigAction tests', () => {
 
 		// ==== OUTPUT PROCESSING ====
 		const displayEvents = spyDisplay.getDisplayEvents();
-		expect(displayEvents).toHaveLength(1);
```

Comment:
Array length is now variable, so we're not checking for an exact length anymore.